src/l1-resources/cloud-outage-simulation/index.ts

```diff
@@ -0,0 +1,180 @@
+// Generated by cdk-import
+import * as cdk from "aws-cdk-lib";
+import * as constructs from "constructs";
+
+/**
+ * An example resource schema demonstrating some basic constructs and validation rules.
```

Comment:
Is this a valid comment? it doesn't seem to be in the `examples` folder.

Reply:
ahh yes! but this comment is an error on the CFN resource, ill fix it there first, if i dont fix it there it will still be showing here